### DISCUSSION TITLE
USA ISM Frequencies, Antennas and rtl_433

### DISCUSSION THREAD

**Discussion by pmaine** [merbanan/rtl_433#3075] [General]
The approved frequencies for the US that interest me are 902 - 928 MHz and 2400 - 2500 MHz
433.05 - 434.79 MHz is not approved in the US but I see lots of devices!
What SDR is supported to listen for devices in the 2.4Ghz range. Also what antennas do you use?

**Comment by zuckschwerdt** [comment]
LimeSDR, e.g. the mini support the 2.4 GHz band. There is a table at https://www.crowdsupply.com/lime-micro/limesdr-mini#comparison-table which also lists HackRF One, Ettus B200, Ettus B210, BladeRF x40 as GHz capable.
